src/main/scala/com/databricks/labs/overwatch/pipeline/WorkflowsTransforms.scala

```diff
@@ -767,147 +768,151 @@ object WorkflowsTransforms extends SparkSessionWrapper {
-        'runId,
-        coalesce('multitaskParentRunId_Started, 'multitaskParentRunId_Completed).cast("long").alias("multitaskParentRunId"),
-        coalesce('parentRunId_Started, 'parentRunId_Completed).cast("long").alias("parentRunId"),
-        coalesce('taskRunId, 'idInJob).cast("long").alias("idInJob"),
-        TransformFunctions.subtractTime(
-          'startTime,
-          array_max(array('completionTime, 'cancellationTime)) // endTS must remain null if still open
-        ).alias("TaskRunTime"), // run launch time until terminal event
-        TransformFunctions.subtractTime(
-          'startTime,
-          array_max(array('completionTime, 'cancellationTime)) // endTS must remain null if still open
-        ).alias("TaskExecutionRunTime"), // from cluster up and run begin until terminal event
-        'run_name,
-        when(coalesce('jobClusterType_Started, 'jobClusterType_Completed).isNull and ('job_clusters.isNotNull or 'new_cluster.isNotNull), "new")
-          .otherwise(coalesce('jobClusterType_Started, 'jobClusterType_Completed)).alias("clusterType"),
-        coalesce('jobTaskType_Started, 'jobTaskType_Completed).alias("taskType"),
-        coalesce('jobTriggerType_Triggered,'jobTriggerType_Started, 'jobTriggerType_Completed, 'jobTriggerType_runNow).alias("jobTriggerType"),
-        when('cancellationRequestId.isNotNull, "Cancelled")
-          .otherwise('jobTerminalState)
-          .alias("terminalState"),
-        'clusterId,
-        'existing_cluster_id,
-        'new_cluster,
-        'tasks.alias("submitRun_tasks"),
-        'job_clusters.alias("submitRun_job_clusters"),
-        'libraries,
-        'access_control_list,
-        'git_source,
-        'manual_override_params,
-        coalesce('workflow_context_runNow, 'workflow_context_submitRun).alias("workflow_context"),
-        'notebook_task,
-        'spark_python_task,
-        'python_wheel_task,
-        'spark_jar_task,
-        'spark_submit_task,
-        'shell_command_task,
-        'pipeline_task,
-        'repairId,
-        'repair_details,
-        struct(
-          'startTime,
-          'submissionTime,
-          'cancellationTime,
-          'completionTime,
-          'timeout_seconds
-        ).alias("timeDetails"),
-        struct(
-          struct(
-            'submitRequestId,
-            'submitResponse,
-            'submitSessionId,
-            'submitSourceIP,
-            'submitUserAgent,
-            'submittedBy
-          ).alias("submissionRequest"),
-          struct(
-            'cancellationRequestId,
-            'cancellationResponse,
-            'cancellationSessionId,
-            'cancellationSourceIP,
-            'cancelledUserAgent,
-            'cancelledBy
-          ).alias("cancellationRequest"),
+  val jobRunsDeriveRunsBase = (etlUntilTime: TimeTypes) => NamedTransformation {
+    (df: DataFrame) => {
+
```

Comment:
what changes we are performing as part of this code change?It looks like the same code above.

Reply:
the actual optimization is in commit https://github.com/databrickslabs/overwatch/pull/1253/commits/d751d5fc75c939892b73f877cb0e5542eb2cc030:
https://github.com/databrickslabs/overwatch/blob/d751d5fc75c939892b73f877cb0e5542eb2cc030/src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala#L271-L274

As of `0820_release` it simply has [this](https://github.com/databrickslabs/overwatch/blame/97236aea0724190459581be950903621d37442bd/src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala#L271):
https://github.com/databrickslabs/overwatch/blob/97236aea0724190459581be950903621d37442bd/src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala#L271

(no call to `.withSparkOverrides()` method)

Everything else is either logging, Spark UI labels, or refactoring that has no impact on performance.

Reply:
resolved by #1228 and #1256